README.md

```diff
@@ -1,2 +1,20 @@
-# hw01
-Repository for first CFSS homework assignment
+# Cheng Yee Lim
+
+![](images/profile_pic.jpg)
+
+> *“A ship is safe in harbor, but that's not what ships are for.”*
+> ― **William G.T. Shedd**
+
+* Singaporean with a severe case of wanderlust
+  + Lived in the United Kingdom for the past three years 
+  + First year in the United States of America
+* Economist by training, but interested to pick up data analytics  
+* To find out more about the coding I do, click [here](www.github.com/limchengyee)
```

Comment:
Use `[here](https://github.com/limchengyee)` instead. This produces [here](https://github.com/limchengyee)